gtars/tests/test.rs

```diff
@@ -0,0 +1,123 @@
+use std::io::{BufRead, BufReader, Read};
+use std::path::{Path, PathBuf};
+use std::fs::{File};
+
+use rstest::*;
+use tempfile::tempdir;
+
+use gtars::uniwig::{parse_bed_file};
+
+#[fixture]
+fn path_to_data() -> &'static str {
+    "tests/data"
+}
+
+#[fixture]
+fn path_to_bed_file() -> &'static str {
+    "tests/data/peaks.bed"
+}
+
+#[fixture]
+fn path_to_sorted_small_bed_file() -> &'static str {
+    "tests/data/test_sorted_small.bed"
+}
+
+#[fixture]
+fn path_to_bed_file_gzipped() -> &'static str {
+    "tests/data/peaks.bed.gz"
+}
+
+mod tests {
+    use std::env::temp_dir;
+    use gtars::uniwig::{Chromosome, read_bed_vec, uniwig_main};
+
+    use super::*;
+
+    #[rstest]
+    fn test_parsed_bed_file(path_to_bed_file: &str) {
+        let path = Path::new(path_to_bed_file);
+        let file = File::open(path).unwrap();
+
+        let mut reader = BufReader::new(file);
+        let first_line = reader.by_ref().lines().next().unwrap().expect("expect");
+        println!("{:?}", first_line);
+
+        let result = parse_bed_file(&first_line);
+
+        if let Some((ctg, st, en)) = result {
+
+            println!("ctg: {}", ctg);
+            println!("st: {}", st);
+            println!("en: {}", en);
+            assert_eq!(st, 7915738);
```

Comment:
Super small thing, but I love to separate large numbers with underscores. So, `7915738`, becomes `7_915_738`. really improves readability.

Reply:
Yeah. This is a good point; I like this approach.